src/main/java/Movie.java

```diff
@@ -1,34 +0,0 @@
-public class Movie {
-    private String title;
-    private String directorName;
-    private String directorSurname;
-    private int lenghtInMinutes;
-    private int productionYear;
```

Comment:
eee, wszystkie zadania finalnie powinny zostać w tym repo :)
Wrzuć je spowrotem, po prostu do jakieogś osobnego pakietu np /zad3/ (czy które tam to było)